lib/pure/os.nim

```diff
@@ -298,8 +298,11 @@ proc setCurrentDir*(newDir: string) {.inline, tags: [].} =
 
 proc expandFilename*(filename: string): string {.rtl, extern: "nos$1",
   tags: [ReadDirEffect].} =
-  ## Returns the full (`absolute`:idx:) path of the file `filename`,
-  ## raises OSError in case of an error.
+  ## Returns the full (`absolute`:idx:) path of an existing file `filename`,
+  ## raises OSError in case of an error. Follows symlinks.
+  ##
+  ## To create absolute paths from any path (existing or not) see
+  ## `<#absolutePath>`_.
```

Comment:
I think ``expandPath`` would make more sense as a name here. ``absolutePath`` makes me wonder how it differs from ``normalizePath``.